flink-runtime/src/main/java/org/apache/flink/runtime/entrypoint/parser/CommandLineOptions.java

```diff
@@ -43,6 +43,42 @@ public class CommandLineOptions {
                     .desc("Port for the rest endpoint and the web UI.")
                     .build();
 
+    public static final Option REPLACE_KEY_VALUE_OPTION =
```

Comment:
Other options are intended to be used by users. It's better to move these new options to a separate place.